include/fiction/layouts/coordinates.hpp

```diff
@@ -665,6 +669,21 @@ struct formatter<fiction::cube::coord_t>
 };
 }  // namespace fmt
 
-#pragma GCC diagnostic pop
+namespace siqad
+{
+using coord_t = fiction::cube::coord_t;
+template <typename CoordinateType>
+CoordinateType constexpr siqad_to_other(const coord_t& coord) noexcept
```

Comment:
I prefer having the `constexpr` qualifier first and then the return type. It doesn't matter, I just want to be consistent